x-pack/plugins/alerting/server/task_runner/rule_loader.ts

```diff
@@ -71,21 +72,30 @@ export function validateRule<Params extends RuleTypeParams>(
   if (!enabled) {
     throw new ErrorWithReason(
       RuleExecutionStatusErrorReasons.Disabled,
-      new Error(`Rule failed to execute because rule ran after it was disabled.`)
+      new Error(`Rule failed to execute because rule ran after it was disabled.`),
+      TaskErrorSource.USER
     );
   }
   alertingEventLogger.setRuleName(rule.name);
   try {
     ruleTypeRegistry.ensureRuleTypeEnabled(rule.alertTypeId);
   } catch (err) {
-    throw new ErrorWithReason(RuleExecutionStatusErrorReasons.License, err);
+    throw new ErrorWithReason(
```

Comment:
whatever we decide to do with disabled connector types, we should do here as well.  I wonder if source `RULE_TYPE` kind of matches what I was describing with "expected" (for disabled connector types) ^^^

Reply:
yes agree, i replaced the above error with `CONNECTOR_TYPE`